src/cycleclock.h

```diff
@@ -79,7 +79,7 @@ inline BENCHMARK_ALWAYS_INLINE int64_t Now() {
   int64_t ret;
   __asm__ volatile("rdtsc" : "=A"(ret));
   return ret;
-#elif defined(__x86_64__) || defined(__amd64__)
+#elif (defined(__x86_64__) || defined(__amd64__)) && !defined(__arm64ec__)
```

Comment:
Is `__x86_64__` or `__amd64__` really defined when `__arm64ec__` is defined?

Reply:
Yes:
https://github.com/llvm/llvm-project/blob/b13bca7387a7aad6eaf3fa1c55bd06fe091f65ed/clang/lib/Basic/Targets/AArch64.cpp#L404-L414

You should think of Arm64EC as "x64 code running on an Arm64 machine with a weird instruction set". So, by default, it needs to use the x64 layout for objects to be compatible with any real x64 code that is linked into the same binary or called across binaries.